### ISSUE TITLE
RBNF missing after upgrading

### ISSUE THREAD

**Issue by juanperi** [elixir-cldr/cldr_numbers#27]
I've updated from 2.25.2 to 2.26.0, and suddenly I started seeing the following error for `nb` locale:
```
iex(1)> IO.inspect(SylarLocalization.Cldr.put_locale("nb"), structs: false)
{:ok,
 %{
   __struct__: Cldr.LanguageTag,
   backend: SylarLocalization.Cldr,
   canonical_locale_name: "nb",
   cldr_locale_name: :nb,
   extensions: %{},
   gettext_locale_name: "nb",
   language: "nb",
   language_subtags: [],
   language_variants: [],
   locale: %{},
   private_use: [],
   rbnf_locale_name: nil,
   requested_locale_name: "nb",
   script: :Latn,
   territory: :NO,
   transform: %{}
 }}
{:ok, #Cldr.LanguageTag<nb [validated]>}
iex(2)> SylarLocalization.Cldr.Number.to_string(1234)
{:error, {Cldr.Rbnf.NotAvailable, "RBNF is not available for the locale :nb"}}
```
This was not happening in the previous version, though, where I would get:
```
iex(1)> IO.inspect(SylarLocalization.Cldr.put_locale("nb"), structs: false)
{:ok,
 %{
   __struct__: Cldr.LanguageTag,
   backend: SylarLocalization.Cldr,
   canonical_locale_name: "nb",
   cldr_locale_name: :nb,
   extensions: %{},
   gettext_locale_name: "nb",
   language: "nb",
   language_subtags: [],
   language_variants: [],
   locale: %{},
   private_use: [],
   rbnf_locale_name: :nb,
   requested_locale_name: "nb",
   script: :Latn,
   territory: :NO,
   transform: %{}
 }}
{:ok, #Cldr.LanguageTag<nb [validated]>}
iex(2)> SylarLocalization.Cldr.Number.to_string(1234)
{:ok, "1¬†234"}
```

Is this a bug? or is it related to the upgrade to release 41 (Update [CLDR](https://cldr.unicode.org) to [release 41] as per the changelog) and it is expected that we might lose access to localized numbers after cldr releases update?

Thank you very much for your help figuring this out!!

**Comment by kipcole9** [comment]
Thanks for the report and sorry for the inconvenience.  Its a bug for sure, and its surfaced because of some changes to the underlying CLDR data.  As of [CLDR-14493](https://unicode-org.atlassian.net/browse/CLDR-14493) the RBNF data for locale "nb" was removed.  This coincided with a change to the locale hierarchy in which "nb" became a child locale to "no" (where "child" is used in the sense of locale inheritance). It seems it has taken quite some time for that change to ripple through the JSON generator that ultimately produces the majority of the data that `ex_cldr` uses.

TLDR; Yes, the RBNF data for "nb" has gone.  However I suspect the intent is that the RBNF data for "no" is intended to be used.

**Is is reasonable, from a language and linguistic point of view, that the RBNF rules for "no" are also appropriate and applicable for "nb"?**

If I assume it is, then most definitely I am not applying any form of inheritance in resolving which RBNF rules to use.  It will be a reasonably straight forward change to do so, but it will take a few days to be able to do that work.
